docs/dev/types_serialization.zh.md

```diff
@@ -24,180 +24,164 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Apache Flink handles data types and serialization in a unique way, containing its own type descriptors,
-generic type extraction, and type serialization framework. This document describes the concepts and the rationale behind them.
+Apache Flink 以其独特的方式来处理数据类型以及序列化，这种方式包括它自身的类型描述符、泛型类型提取以及类型序列化框架。
+本文档描述了它们背后的概念和基本原理。
 
 * This will be replaced by the TOC
 {:toc}
 
 
-## Type handling in Flink
+## Flink 中的类型处理
 
-Flink tries to infer a lot of information about the data types that are exchanged and stored during the distributed computation.
-Think about it like a database that infers the schema of tables. In most cases, Flink infers all necessary information seamlessly
-by itself. Having the type information allows Flink to do some cool things:
+Flink 对分布式计算中发生的数据交换以及排序，试图推断有关数据类型的大量信息。
+可以把它想象成一个推断表结构的数据库。在大多数情况下，Flink 可以依赖自身透明的推断出所有需要的类型信息。
+掌握这些类型信息可以帮助 Flink 实现很多意想不到的特性：
 
-* Using POJOs types and grouping / joining / aggregating them by referring to field names (like `dataSet.keyBy("username")`).
-  The type information allows Flink to check (for typos and type compatibility) early rather than failing later at runtime.
+* 对于使用 POJOs 类型的数据，可以通过指定字段名（比如 `dataSet.keyBy("username")` ）进行 grouping 、joining、aggregating 操作。
```

Comment:
进行 grouping 、joining、aggregating 操作 -> 进行 grouping / joining / aggregating 操作 ?

Reply:
The punctuation ”/ “ in English , I think is the same as the punctuation '、' in Chinese. Because there is not '、' in English ,usually using ',' instead .  The meaning of the punctuation ”/ “ in this docs may be 'or' , so I thank '、' is more proper.   https://www.zhihu.com/question/24518171 may be helpfull